src/App.tsx

```diff
@@ -63,6 +84,37 @@ const checkoutTheme = createTheme({
         },
       },
     },
+    MuiOutlinedInput: {
+      styleOverrides: {
+        root: {
+          borderRadius: "8px",
+        },
+      },
+    },
+    MuiLoadingButton: {
+      styleOverrides: {
+        loadingIndicator: {
+          color: "white !important",
+        },
+        root: ({ ownerState }) =>
```

Comment:
You can avoid using important and the conditionals with "&.MuiLoadingButton-loading" as a subkey of root instead of using the ownerState.

Reply:
Also it seems that the button text should not be transformed to capitalized anymore